README.md

```diff
@@ -13,6 +13,10 @@ This repository provides demonstrations for:
     + predefined directory lookup methods (see [workflow_miniscope/paths.py](workflow_miniscope/paths.py))
 3. Ingestion of clustering results (built-in routine from the imaging element)
 
++ This workflow provides support for miniscope calcium imaging data acquired with the `Miniscope-DAQ` systems.  And for data processed with `Suite2p`, `CaImAn`, or `Miniscope Analysis`.
+    + Note the [`Miniscope Analysis`](https://github.com/etterguillaume/MiniscopeAnalysis) package is not currently maintained.
```

Comment:
A naive question here: why do we support the Miniscope-DAQ-V3 and MiniscopeAnalysis in the workflow not in the element?

Reply:
Good question.  `Miniscope-DAQ-V3` and `MiniscopeAnalysis` are supported in both the `workflow` and `element`.  Metadata from `Miniscope-DAQ-V3` is ingested in the `scan` schema, and processed data from `MiniscopeAnalysis` is ingested in the `imaging` schema.